stage1/init/pod.go

```diff
@@ -430,7 +431,7 @@ func (p *Pod) writeEnvFile(env types.Environment, appName types.ACName, privateU
 
 // PodToSystemd creates the appropriate systemd service unit files for
 // all the constituent apps of the Pod
-func (p *Pod) PodToSystemd(interactive bool, flavor string, privateUsers string) error {
+func (p *Pod) PodToSystemd(interactive bool, flavor string, privateUsers string, n *networking.Networking) error {
```

Comment:
It's awkward that last parameter is only used if `flavor == "kvm"`. Is there a way to work around that by maybe creating a separate function?